src/Dofs/DofHandler.jl

```diff
@@ -137,7 +137,7 @@ function celldofs(dh::AbstractDofHandler, i::Int)
     return celldofs!(zeros(Int, ndofs_per_cell(dh, i)), dh, i)
 end
 
-#TODO: perspectively remove in favor of `getcoordinates!(global_coords, grid, i)`?
+#TODO: perspectively remove in favor of `get_cell_coordinates!(global_coords, grid, i)`?
 function cellcoords!(global_coords::Vector{Vec{dim,T}}, dh::DofHandler, i::Union{Int, <:AbstractCell}) where {dim,T}
     cellcoords!(global_coords, dh.grid, i)
 end
```

Comment:
Should this also be called `get_cell_coordinates!`?

Reply:
I wasn't sure, but I think it makes sense.

But this does introduce a new method, another option is to simply remove it all together and require users to pass a grid if they want grid info.
And with your PR on `getgrid(dh)` that should be easy...

Reply:
Removing this will be breaking. I do not think it is bad to have this convenience function available, because it is a common pattern in our assembly loops to call stuff on the dof handler/field handler and get the correct stuff. What do you think?

Reply:
Yes, I believe that is the intention. 
So IMO it is a design choice: Should one expect grid-api functions to also by giving a dofhandler instead of a grid. 
In that case, there is a bunch of other functions that should accept `dh` input as well. 
I think it would be good to be consistent here, so rethinking this I will deprecate `cellcoords!(x,dh,args...)` here, as I think that design discussion should take place separately.